src/utils/cspUtils.js

```diff
@@ -21,7 +21,11 @@ function toRegExp(string) {
     "\\*",
     "[a-z0-9-]*"
   ) // convert wildcards
-  const RegExpString = RegExp(wildcardEscapedStrippedString) // converts to RegExp
+  const relativeWildcardEscapedStrippedString = wildcardEscapedStrippedString.replace(
+    "https://",
+    "^(https:)?//"
+  ) // support protocol-relative URLs without supporting http://
```

Comment:
For `//something.com` to resolve to `http://something.com`, the user will have to be browsing the page via a `http://` page. I think that is okay because (1) I'm assuming all Isomer sites are in https and (2) the source will redirect to the https version if possible.

I added this line of code so that when the user copy-pastes the embed code inside the Markdown editor, no additional changes are necessary. The embed code comes with the source URL being `//www.instagram.com/embed.js`.

Reply:
ok so let me surmise my understanding so far just to ensure we're on the same page!

1. protocol relative URLs resolve relative to the user agent
2. isomer enforces all sites to be on **https**
3. by extension, **all** protocol relative URLs will hence resolve to **https://** as the user agent will be on `https://` whilst viewing isomer sites